src/custom/hooks/useMarkdown.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+
+export default function useMarkdown(file: string) {
+  const [content, setContent] = useState('')
+
+  useEffect(() => {
+    const fetchContent = async () => {
+      await fetch(file)
+        .then(res => res.text())
+        .then(text => {
+          setContent(text)
+        })
+        .catch(error => {
+          console.log(`Error fetching markdown content file ${file}: `, error)
```

Comment:
You need to add error handling. 
You could expose the error message out of the hook. You would need another useState

Reply:
Right now if `error` is not `null` it returns an Error in the form of a `string`. If `error` is `null` it returns the file content (as a string).